frontend/src/components/explorer/gemBrowser/GemSearch.vue

```diff
@@ -292,23 +290,21 @@ export default {
       display: none;
     }
 
-    .searchResultSection{
-      > a {
+    .searchResultSection {
+      > div {
         display: block;
         padding: 7px 0px;
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
       }
 
-      .search-result-icons a {
-        &:first-child {
-          color: $primary;
-        }
+      .color-gb {
+        color: $primary;
+      }
```

Comment:
It looks like this could be replaced by `is-primary` or something of the sort.

Reply:
I was trying to keep the styling the same as before. The color of the Gem-Browser icon is not the same by using `is-primary`.

Reply:
This sounds very confusing to me - do you mean `is-primary` shows a different color than `color: $primary`?

Reply:
yes

Reply:
Right, the correct Bulma class for this context is `has-text-primary`, which renders the expect `primary` color.

Reply:
Great to learn the syntax of Bulma for setting the text color. While `has-text-primary` does the trick, `has-text-icon-interaction-partner` does not seem to work, although `icon-interaction-partner` is also defined in `frontend/src/style/vars.scss`. Nevertheless, I do think the code is more understandable with the current setting, i.e. using `color-gb` and `color-ip`. In addition, with this setting, if the color of icons needs to be changed, we only need to change the css without touching the template.

Reply:
[`has-text-primary`](https://bulma.io/documentation/helpers/color-helpers/ is a Bulma class. By default, we should use Bulma classes as much as possible.

It's no surprise that there is no `has-text-icon-interaction-partner`, since these classes are not dynamically defined. The usage of `vars.scss` is just to share or overload the Bulma-provided CSS variables. Feel free to define a global class in `App.vue` for this class.

Reply:
Good to know. I've implemented the usage of Bulma syntax in the commit 94e4fe22